source/features/user-profile-follower-badge.tsx

```diff
@@ -1,17 +1,26 @@
 import './user-profile-follower-badge.css';
 import React from 'dom-chef';
 import select from 'select-dom';
+import cache from 'webext-storage-cache';
 import * as api from '../libs/api';
 import features from '../libs/features';
 import {getUsername, getCleanPathname} from '../libs/utils';
 
-async function init(): Promise<void> {
+const userIsAFollower = cache.function(async (): Promise<boolean> => {
 	const {httpStatus} = await api.v3(
 		`users/${getCleanPathname()}/following/${getUsername()}`,
 		{ignoreHTTPStatus: true}
 	);
 
-	if (httpStatus === 204) {
+	return httpStatus === 204;
+}, {
+	maxAge: 3,
+	staleWhileRevalidate: 20,
+	cacheKey: () => __featureName__ + ':' + getCleanPathname()
```

Comment:
Let's start avoiding `__featureName__` here

```suggestion
	cacheKey: ([userA, userB]) => 'user-follows:${userA}:${userB}'
```